unabridged-works-volume-3/original/15-letters.adoc

```diff
@@ -3865,14 +4051,17 @@ nor comfort.
 
 Thy friend, in the truest, sincerest, and most constant love.
 
+[.signed-section-signature]
 I+++.+++ P.
 
+[.signed-section-context-close]
 London, 1st of Ninth Month, 1675
 
-TO CATHARINE PORDAGE
+[.letter-heading]
+To Catherine Pordage
 
-AH!
-MY POOR, DISTRESSED, ENTANGLED FRIEND,
+[.salutation]
+Ah! My Poor, Distressed, Entangled Friend,
```

Comment:
This one is a bit tough. I think I agree with keeping it a `.salutation`, but the title-casing feels weird to me... I think let's change it to:

```ASCIIDOC
[.salutation]
Ah! My poor, distressed, entangled friend,
```